src/components/_overlays/ContextMenu/ContextMenu.tsx

```diff
@@ -12,12 +14,7 @@ export type MenuItemProps = {
 }
 
 export const ContextMenuItem: React.FC<MenuItemProps> = React.memo(({ icon, onClick, title }) => {
```

Comment:
I don't think we need this component anymore, seems like we could use `ListItem` directly in `ContextMenu` component